app/uk/gov/hmrc/platformstatusbackend/services/StatusChecker.scala

```diff
@@ -76,4 +89,31 @@ class StatusChecker @Inject()(appConfig: AppConfig) {
     } yield result
   }
 
+  private def checkDesHealthcheck(appConfig: AppConfig)(implicit executionContext: ExecutionContext, futures: Futures, hc: HeaderCarrier): Future[PlatformStatus] = {
+    val requestTimestamp = LocalDateTime.now(ZoneOffset.UTC).truncatedTo(ChronoUnit.SECONDS).toString
```

Comment:
This will create a string with no timezone, and requires other parties to understand that it is a local date time relative to UTC. Shouldn't we just be using `Instant` for absolute times?

Reply:
Based on the specification for the endpoint, I'm not 100% what will happen if we provide 'Z' at the end.
Will update the code, and give it a go though.